src/components/CircularButton.tsx

```diff
@@ -5,18 +5,22 @@ type Direction = "left" | "top" | "right";
 export interface ICircularButtonProps {
   direction?: Direction;
   size?: number;
+  className?: string;
+  handlerOnClick?: (newVal?) => void;
 }
-
 const CircularButton: FunctionComponent<ICircularButtonProps> = (props) => {
   return (
-    <button className="circular-button">
+    <button
+      className="circular-button"
+      onClick={(newVal) => props.handlerOnClick(newVal)}
+    >
       <Icon
         type="arrow"
         boxSize={"0 0 28 28"}
         size={28}
-        className={`icon ${props.direction === "top" ? "direction-top" : ""}${
+        className={`icon ${props.direction === "top" ? "direction-top" : ""} ${
           props.direction === "left" ? "direction-left" : ""
-        }`}
+        } ${props.className}`}

```

Comment:
This class should be applied to element "button" or "icon"?

Reply:
- [ ] [Done ](https://github.com/AnastasiSo/Portfolio-Project/pull/16/files#r1117074027).